### ISSUE TITLE
Netlify adapter does not respect entry.ssr settings

### ISSUE THREAD

**Issue by zanettin** [BuilderIO/qwik#1704]
### Qwik Version

0.10.0

### Operating System (or Browser)

OSX 12.6 / any browser

### Node Version (if applicable)

16.13.0

### Which component is affected?

Qwik Runtime

### Expected Behaviour

html `lang` attribute is also present when using netlify adapter

### Actual Behaviour

html `lang` attribute is missing when netlify adapter is present

### Additional Information

steps to repro:
1. create a new app
2. add netlify adapter
3. open `entry.ssr.tsx` file
4. change content to:
```
export default function (opts: RenderToStreamOptions) {
  return renderToStream(<Root />, {
    manifest,
    containerAttributes: {
      lang: "de",
    },
    ...opts,
    prefetchStrategy: {
      implementation: {
        linkInsert: null,
        workerFetchInsert: null,
        prefetchEvent: "always",
      },
    },
  });
}
```
5. run `npm run build` and inspect output in the netlify directory

**Comment by zanettin** [comment]
tested the node adapter which is working as expected 👍

**Comment by zanettin** [comment]
`npm run preview` does not work because of #1393

**Comment by zanettin** [comment]
solved with `0.11.1`
